.ddev/config.yaml

```diff
@@ -0,0 +1,183 @@
+name: meals
+type: php
+docroot: web
+php_version: "5.6"
+webserver_type: apache-fpm
+router_http_port: "80"
+router_https_port: "443"
+xdebug_enabled: false
+additional_hostnames: []
+additional_fqdns:
+  - meals.test
+mariadb_version: ""
+mysql_version: "5.7"
+use_dns_when_possible: true
+composer_version: "1"
+web_environment:
+  - FINAL_RELEASEFOLDER=/var/www/html
+hooks:
+  post-start:
+    - exec: "cd app/Resources && bundle install && yarn && yarn build"
+    - exec: "app/console assets:install web --symlink --relative"
+    - exec: "setup/clear_cache.sh"
+
+# This config.yaml was created with ddev version v1.17.5
```

Comment:
are this comments required?

Reply:
We can remove them, but I liked this kind of "inline documentation" which ddev created. :)